test/test_transforms_tensor.py

```diff
@@ -412,19 +410,15 @@ def test_resize_scripted(self, dt, size, max_size, interpolation, device):
             # This is a trivial cast to float of uint8 data to test all cases
             tensor = tensor.to(dt)
         if max_size is not None and len(size) != 1:
-            pytest.xfail("with max_size, size must be a sequence with 2 elements")
+            pytest.skip("Size should be an int or a sequence of length 1 if max_size is specified")
```

Comment:
Question: Why skip instead of xfail?

Reply:
That's a good question :)

According to https://docs.pytest.org/en/stable/how-to/skipping.html
We skip in case windows tests on non-windows platform and xfail if we expect a test to fail for some reason.

In our case, the configuration "max_size is not None and len(size) != 1" can not be tested as we explicitly raise and error. Proper solution to that is to catch the error and check the message text with `pytest.raises`.

I felt like skipping is more appropriate than xfail. But it is a matter of taste if you think it is better to revert to xfail I can do that and will only fix the message which is incorrect

Reply:
I was expecting a pytest.raises where if I'm honest but no strong opinions.